package.json

```diff
@@ -51,17 +51,17 @@
     "create:theme": "node buildutils/lib/create-theme.js",
     "deduplicate": "jlpm yarn-deduplicate -s fewer --fail",
     "docs": "lerna run docs",
-    "eslint": "eslint --fix .",
-    "eslint:check": "eslint .",
+    "eslint": "eslint --ext .js,.jsx,.ts,.tsx --fix .",
+    "eslint:check": "eslint --ext .js,.jsx,.ts,.tsx .",
     "get:dependency": "node buildutils/lib/get-dependency.js",
     "postinstall": "node scripts/ensure-buildutils.js",
     "integrity": "node scripts/ensure-buildutils.js && node buildutils/lib/ensure-repo.js",
     "lighthouse": "lighthouse http://localhost:8888/ --chrome-flags='--headless' --emulated-form-factor=none --throttling-method=provided --only-categories performance",
     "lighthouse:compare": "node testutils/lib/compare-lighthouse.js",
     "lighthouse:throttling:start": "comcast --latency=40 --target-bw=30000 --packet-loss=0.2%",
     "lighthouse:throttling:stop": "comcast --stop",
-    "lint": "jlpm && jlpm run prettier && jlpm run eslint && jlpm run tslint",
-    "lint:check": "jlpm run prettier:check && jlpm run eslint:check && jlpm run tslint:check",
+    "lint": "jlpm && jlpm run eslint && jlpm run prettier",
```

Comment:
why swap the run order of prettier and eslint, iirc running prettier first is the standard

Reply:
eslint was undoing some of the things prettier was doing with a previous set of rules.  I'll revert this change.  Cheers!